voltronerability-report.md

```diff
@@ -0,0 +1,34 @@
+#Voltronerability Report
+Authors Paul and AJ
+Date: April 26
+
+##Voltronerability 1: Injection
+###Exposure
+We found an instance of vulnerability to injection by typing the following into the "new image" field: testPic',(select password from auth_user where username='admin'),8);--.  Clicking the picture after that point provided a hashed version of the admin credentials.
+###Repair
+We no longer inject sql directly into the database, but instead we use the ORMs built in features to sanitize the inputs prior to entry.
+##Voltronerability 2: Broken Auth
+###Exposure
+We were able to set the is_admin field by manually editing the new user form and swapping the 'email' spot for is_admin.  We were also able to do the same thing for is_superuser.
```

Comment:
So you made "is_admin" and "is_superuser" form input fields out of a normal text input? Show me the code.